examples/lighting-app/nrfconnect/README.md

```diff
@@ -1,289 +1,333 @@
+The nRF Connect Lighting Example demonstrates how to remotely control a white non-dimmable
+light bulb.
+
+The example is based on [CHIP](https://github.com/project-chip/connectedhomeip) and the nRF Connect
+platform, and supports remote access and control of a lighting over a low-power, 802.15.4 Thread network.
+It is capable of being paired into an existing CHIP network along with other CHIP-enabled devices.
+
+You can use this example as a reference for creating your own application
+that uses buttons and LEDs to change the lighting and device states.
+
+<hr>
+
+-   [Overview](#overview)
+    -   [Bluetooth LE Advertising](#bluetooth-le-advertising)
+    -   [Bluetooth LE Rendezvous](#bluetooth-le-rendezvous)
+    -   [Thread provisioning](#thread-provisioning)
+-   [Requirements](#requirements)
+-   [Device UI](#device-ui)
+-   [Setting up the environment](#setting-up-the-environment)
+    -   [Using Docker container for setup](#using-docker-container-for-setup)
+    -   [Using native shell for setup](#using-native-shell-for-setup)
+-   [Building](#building)
+-   [Configuring the example](#configuring-the-example)
+-   [Flashing and debugging](#flashing-and-debugging)
+-   [Testing the example](#testing-the-example)
 
 <hr>
 
--   [CHIP nRF52840 Lighting Example Application](#chip-nrf52840-lighting-example-application)
-    -   [Introduction](#introduction)
-    -   [Device UI](#device-ui)
-    -   [Building](#building)
-        -   [Using Docker container](#using-docker-container)
-        -   [Using Native shell](#using-native-shell)
-        -   [Supported nRF Connect SDK versions](#supported-nrf-connect-sdk-versions)
-    -   [Configuring the example](#configuring-the-example)
-    -   [Flashing and debugging](#flashing-and-debugging)
-    -   [Accessing the command line](#accessing-the-command-line)
+<a name="overview"></a>
+
+## Overview
+
+This example is running on the nRF Connect platform, which is based
+on the [nRF Connect SDK](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/index.html)
+and [Zephyr RTOS](https://zephyrproject.org/). Visit CHIP's
+[nRF Connect Platform Overview](TODO:...) to read more information about
+platform structure and dependencies.
+
+The CHIP device that runs the lighting application is controlled by the CHIP controller
+device over the Thread protocol. By default, the CHIP device should have Thread disabled,
+and it should be paired with CHIP controller and get configuration from it.
+Some actions required before establishing full communication are described below.
+
+The example also comes with a test mode, which allows to start Thread with the default settings
+by pressing button manually. However, this mode does not guarantee that the device will be able
+to communicate with the CHIP controller and other devices.
+
+### Bluetooth LE Advertising
+
+After powering up the device for the first time, it will start advertising over
```

Comment:
"For security reasons, BLE advertising won't start automatically after powering up the device. To make the device discoverable, a user must press **Button 4**."?

Reply:
To me saying that the application should start advertising, but it will not suggests that something doesn't work as designed :). It is a user who should make an action, not the app. Right?

Reply:
"should" would make me enter a loop of waiting for action that never happens. Damian's suggestion is good, just don't use "BLE" (should be "Bluetooth LE") and "a user must press" ("press" is enough).